app/views/evidence/_form.html.erb

```diff
@@ -20,7 +20,7 @@
         data: {
           allow_dropdown: @evidence.new_record?,
           behavior: 'auto-save rich-toolbar drop-zone',
-          paths: editor_paths,
+          paths: editor_paths.merge!(@form_preview_path),
```

Comment:
Can we directly use `@form_preview_path` in the MarkupHelper instead of messing with the view? That way, we have all the paths logic in one place: `editor_paths`